Index: work_moved_flask-babel-0.12.2-py_0_noarch/flask_babel/__init__.py
===================================================================
--- work_moved_flask-babel-0.12.2-py_0_noarch.orig/flask_babel/__init__.py
+++ work_moved_flask-babel-0.12.2-py_0_noarch/flask_babel/__init__.py
@@ -16,7 +16,11 @@ from contextlib import contextmanager
 from flask import current_app, request
 from flask.ctx import has_request_context
 from babel import dates, numbers, support, Locale
-from werkzeug import ImmutableDict
+try:
+    from werkzeug import ImutableDict
+except ImportError:
+    from werkzeug.datastructures import ImmutableDict
+
 try:
     from pytz.gae import pytz
 except ImportError:
